terraform/main.tf

```diff
@@ -16,7 +16,7 @@ data "google_client_openid_userinfo" "me" {
 
 resource "google_bigquery_dataset" "dataset" {
   dataset_id                  = var.bq_ops_dataset
-  location                    = var.location
+  location                    = var.region
```

Comment:
What's the reason for the name mismatch here?

Reply:
Other than US and EU, location and region are the same. Location is often used for multi-locations deployment of resources and can be also set for a single region.